hack/conformance/check_conformance_test_requirements.go

```diff
@@ -48,7 +48,7 @@ func checkAllProviders(e2eFile string) error {
 
 	fileInput, err := ioutil.ReadFile(e2eFile)
 	if err != nil {
-		return errors.Wrapf(err, "Failed to read file %s", e2eFile)
+		return fmt.Errorf("Failed to read file %s:%w", e2eFile, err)
```

Comment:
https://blog.golang.org/go1.13-errors suggests there should be `%s: %w` (space before %w)

Reply:
Thanks. I have fixed it.